app/tests/utils/test_run_execution.py

```diff
@@ -25,6 +25,7 @@
 from app.schemas import TestSelection
 from app.schemas.test_run_execution import TestRunExecutionCreate
 from app.tests.utils.project import create_random_project
+from app.tests.utils.utils import default_matter_config
```

Comment:
Should these general test cases use a matter specific config? I noticed that the config mocks for matter specific negative testing was only used in test cases inside the program's folder. Maybe the mocks should be declared in there too?
I suggest having a mocked default_config in `app.tests.utils.utils` that doesn't have matter stuff and migrating these mocks to the Matter program folder.

Reply:
I agree with your suggestion, but in an initial analysis a big amount of work is required, so we need to plan this task,